torchgeo/samplers/batch.py

```diff
@@ -70,7 +70,7 @@ def __init__(
         dataset: GeoDataset,
         size: Union[Tuple[float, float], float],
         batch_size: int,
-        length: int,
+        length: Optional[int] = None,
```

Comment:
Should we specify in the docstring what this defaults to?

Reply:
It's a bit hard to describe exactly what this defaults to. Any suggestions for wording?

Reply:
Ping @calebrob6

Reply:
It appears to default to the maximal number of non-overlapping patches of size `size` that can be fit into the area of the dataset.